R/find_exports.R

```diff
@@ -42,14 +46,21 @@ extract_meta <- function(lns) {
       glue_collapse(lns, sep = "\n  "),
       1, 80
     )
-    ui_throw(
-      "Rust code contains invalid attribute macros.",
-      c(
-        bullet_x("No valid {.code fn} or {.code impl} block found in the \\
-          following sample:"),
-        code_sample
-      )
+
+
+    cli::cli_abort(
```

Comment:
I couldn't figure out a way to maintain the formatting the cargo errors. What I had tried collapsed new lines. I can try and find a simpler way

Reply:
It's more that I thought `cli_fmt()` just returned a string that would be identical to what `cli_abort()` would output here.

Reply:
Ah yes I see. `{.code ...}` in dot points like `cli_abort` is inline and has no newlines. I would suggest replacing the outermost `cli_abort` with `stop()` because it's doing the exact same thing here I believe.

Reply:
This would have to be `rlang::abort()` instead of `stop()` because stop does not provide you the ability to set the error class

Reply:
Indeed! `(>= 1.0.5)`